tests/test_model_saving.py

```diff
@@ -307,6 +306,30 @@ def test_loading_weights_by_name_and_reshape():
             if old_weights[i]:
                 assert_allclose(old_weights[i][j], new_weights[j], atol=1e-05)
 
+    # delete and recreate model with `use_bias=False`
+    del(model)
+    model = Sequential()
+    model.add(Conv2D(2, (1, 1), input_shape=(1, 1, 1), use_bias=False, name='rick'))
+    model.add(Flatten())
+    model.add(Dense(3, name='morty'))
+    with pytest.raises(ValueError):
```

Comment:
When testing for errors, prefer using a regex (or `substring is in msg` check) to verify that the *expected* error is being raised. Occasionally we've had situations where an exception test would pass because it was encountering a different error (of the same type).

Reply:
Thank you for the comment, @fchollet. Updated.